src/components/Modal/Modal.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import InfoIcon from '@mui/icons-material/Info';
+import './Modal.css';
+import wordle01 from './wordlesolver01.png';
+import wordle02 from './wordlesolver02.png';
+import wordle03 from './wordlesolver03.png';
+
+export default function Modal({ modal, setModal }) {
+
+    const toggleModal = () => {
+        setModal(!modal);
+    };
+
+    if (modal) {
+        document.body.classList.add('active-modal');
+    } else {
+        document.body.classList.remove('active-modal');
+    }
+
+    const modalRender = () => {
```

Comment:
This should probably just be a component (pull it out of this render loop) and you don't have to render it if modal is false